tests/unit/s2n_self_talk_psk_test.c

```diff
@@ -142,6 +142,11 @@ static int s2n_test_select_psk_identity_callback(struct s2n_connection *conn, vo
     return S2N_SUCCESS;
 }
 
+int static s2n_client_hello_no_op_cb(struct s2n_connection *conn, void *ctx)
+{
```

Comment:
TIL you can do "int static" instead of "static int": https://stackoverflow.com/questions/45296414/order-of-static-function-keyword-vs-return-type-specifier-in-c But as that discussion points out, not a good idea. You should flip it back.

Reply:
heh whoops

Reply:
I am curious if this would pass our CI, but I fixed it.